test/framework/options.py

```diff
@@ -4800,25 +4675,42 @@ def test_github_new_update_pr(self):
         ]
         self._assert_regexs(regexs, txt)
 
+        # Commit message must not be specified for only new ECs
+        args_new_pr = args + ['--pr-commit-msg=just a test']
+        error_msg = r"PR commit msg \(--pr-commit-msg\) should not be used"
+        with self.mocked_stdout_stderr():
+            self.assertErrorRegex(EasyBuildError, error_msg, self.eb_main, args_new_pr, raise_error=True, testing=False)
+
+        # But commit message can still be specified when using --force
+        args_new_pr.append('--force')
+        txt, _ = self._run_mock_eb(args_new_pr, do_build=True, raise_error=True, testing=False)
+        regexs_with_msg = [
+            error_msg,  # Still shown as a warning
+            r'== Using the specified --pr-commit-msg',
+            r'\* title: "just a test"',
+        ]
+        self._assert_regexs(regexs_with_msg, txt)
+
         # add unstaged file to git working dir, to check on later
         unstaged_file = os.path.join('easybuild-easyconfigs', 'easybuild', 'easyconfigs', 'test.eb')
         write_file(os.path.join(git_working_dir, unstaged_file), 'test123')
+        # Remove other temporary git working dirs
+        res = glob.glob(os.path.join(self.test_prefix, 'eb-*', 'eb-*', 'git-working-dir*'))
+        res = [d for d in res if d != git_working_dir]
+        for d in res:
+            remove_dir(d)
```

Comment:
Don't use single-letter variables outside of list comprehensions:
```suggestion
        for path in res:
            remove_dir(path)